### PR TITLE
Configure lib dir to be loaded by default

### PR DESCRIPTION
**Pull request by patbenatar**
@cltatman

### CODE REVIEW

File: lib/suspenders/app_builder.rb [hunk 1]
```diff
@@ -571,6 +571,15 @@ def configure_quiet_assets
       inject_into_class "config/application.rb", "Application", config
     end
 
+    def configure_lib_directory
+      config = <<-RUBY
+    config.autoload_paths << Rails.root.join('lib')
+    config.eager_load_paths << Rails.root.join('lib')
```
Comment (cltatman) [inline lib/suspenders/app_builder.rb:577]: i don't normally do this `eager_load_paths` when i'm adding `autoload_paths`, should i?

Comment (patbenatar) [reply lib/suspenders/app_builder.rb:577]: In Rails5, yeah http://stackoverflow.com/questions/38198668/rails-5-load-lib-files-in-production

Comment (cltatman) [reply lib/suspenders/app_builder.rb:577]: interesting!

**Comment by cltatman** [comment]
@patbenatar :shipit:

**Pull request merged**